### PR TITLE
jupyterlab-lsp 3.4.1

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by bollwyvl** [comment]
@conda-forge-admin please rerender

**Comment by bollwyvl** [comment]
@krassowski sorry it took me a while to get around to this, fabulous work getting this out so quickly, and the broader contribution to the ecosystem docs, etc. i realize now we _could_ add the source build test here, for giggles... though if we get all the way to this point, it's _awfully_ far down the line on any given sunday...

**Comment by krassowski** [comment]
Thanks for adding the extra check!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# this is the version the bot will _probably_ pick up and PR.
-{% set version = "3.4.0" %}
+# this is the version the bot will _hopefully_ pick up and PR.
```
Comment (krassowski) [inline recipe/meta.yaml:1]: _ultimately, but probably too late_ ;)

**Pull request merged**